python/lsst/analysis/drp/histPlot.py

```diff
@@ -237,20 +199,20 @@ def run(self, catPlot, dataId, runName, skymap, tableName, bands, plotName):
         # Get useful information about the plot
         plotInfo = parsePlotInfo(dataId, runName, tableName, bands, plotName, SN)
         # Calculate the corners of the patches and some associated stats
-        sumStats = generateSummaryStats(plotDf, self.config.summaryStatsLabel, skymap, plotInfo)
+        sumStats = generateSummaryStats(datPlot, self.config.summaryStatsColumn, skymap, plotInfo)
```

Comment:
Why did the name get changed to datPlot? It would be nice to have consistency across all the plot types.

Reply:
Changed back to `plotDf`.